src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -104,7 +105,12 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
             if (attrib == null)
                 return;
 
-            SelectedIndex = ParseSelectedIndex(attrib.Value, Items);
+            selectedIndex = ParseSelectedIndex(attrib.Value, Items);
+
+            if (selectedIndex < 0)
+            {
+                Warning(Resources.NothingIsSelectedWarning);
+            }
         }
```

Comment:
This is fixes for tests from today as well as the refactoring of the StartConfigurations.